### PR TITLE
Modernize content

### PR DESCRIPTION
**Pull request by marcoroth**
This pull request modernizes a few things:

* [`stimulus` -> `@hotwired/stimulus`](https://github.com/julianrubisch/better-stimulus/commit/7a8d66a040edd57e6934665dc88aaab0042e7669)
* [`stimulusjs.org` -> `stimulus.hotwired.dev`](https://github.com/julianrubisch/better-stimulus/commit/6153cad57867a643e42e2766588957b2ea9f4c2c)
* [`Turbolinks` -> `Turbo`](https://github.com/julianrubisch/better-stimulus/commit/39b49c06b77f7495a17ded11d250a3cfef815a90)

### CODE REVIEW

**Comment by julianrubisch** [comment]
Wow, thanks! Will go over it soon.

File: rails-ujs/submit-form.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Works
 
 ```js
 import Rails from "@rails/ujs";
```
Comment (julianrubisch) [inline rails-ujs/submit-form.md:43]: I should probably redo the whole category with @rails/requestjs 🙈

Comment (marcoroth) [reply rails-ujs/submit-form.md:43]: Oh right, yeah. I missed that part

Comment (julianrubisch) [reply rails-ujs/submit-form.md:43]: nevermind, that's a whole different story

**Pull request merged**